scripts/mobile/ios-repack.sh

```diff
@@ -109,7 +109,7 @@ TARGET="$(cd "$(dirname "${SOURCE}")"; pwd)/${APPLICATION}.repack"
     "${SECURITY}" cms -D -i "${PROFILE}" > "${PROVISION}"
     "${PLISTBUDDY}" -x -c 'Print :Entitlements' "${PROVISION}" > "${ENTITLEMENT}"
 
-    EXENAME=$(defaults read ${BUILD}/Payload/${APPLICATION}.app/Info.plist CFBundleExecutable)
+    EXENAME=$(defaults read ${BUILD}/Payload/"${APPLICATION}".app/Info.plist CFBundleExecutable)
```

Comment:
I would expect the entire path being wrapped by " ?
E.g. if the BUILD also contains spaces